packages/components/src/components/List/UnorderedList.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from "@xstyled/styled-components";
+import React, { ReactElement } from "react";
+import { Box } from "../../primitives/Box";
+import { ListItemProps } from "./ListItem";
+
+export type ListFontColors = "colorText" | "colorTextLink";
+export interface UnorderedListProps
+  extends Omit<React.HTMLAttributes<HTMLUListElement>, "color"> {
+  /** Changes the color of the list items. */
+  color?: ListFontColors;
+
+  /** The list items */
+  children: ReactElement<ListItemProps> | ReactElement<ListItemProps>[];
+}
+
+/** A list of items with bullet points */
+const InnerUnorderedList = styled(Box.ul)`
+  margin-left: space30;
+
+  li {
+    position: relative;
+    padding-left: space30;
+
+    &::before {
+      content: "•";
```

Comment:
Why using just `ul` is not enough to add the bullet points?

Reply:
The bullet points have their spacing and size defined by the font size. In our case, it is really small, so we can't control it. It is possible to control the size using the pseudo element `::marker`, but not the spacing.